src/lib/import_export/binary/binary_parser.cpp

```diff
@@ -28,7 +28,7 @@ std::shared_ptr<Table> BinaryParser::parse(const std::string& filename) {
 
   auto [table, chunk_count] = _read_header(file);
   for (ChunkID chunk_id{0}; chunk_id < chunk_count; ++chunk_id) {
-    _import_chunk(file, table);
+    _import_chunk(file, table, chunk_id < (chunk_count - 1));
```

Comment:
I'd prefer for `chunk_id < (chunk_count - 1)` to be in its own variable first. When reading this, it is hard to guess what it means

Reply:
Will change once we decide for or against finalization.